SwiftUIHomeworks/SwiftUIHomeworks/ViewModels/GreetingViewModel.swift

```diff
@@ -0,0 +1,35 @@
+// GreetingViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Greeting view model
+final class GreetingViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let urls = [
+            "https://i.postimg.cc/nr6RgPGW/icons8-sofa-100.png",
+            "https://i.postimg.cc/htSvCfTR/bedRoom.png",
+            "https://i.postimg.cc/mkLQMMCX/buffet.png",
+            "https://i.postimg.cc/B6G5SR6G/icons8-coffee-table-100.png",
+            "https://i.postimg.cc/MpNb4nJH/schkaf.png",
+            "https://i.postimg.cc/s2W9R33c/stul.png",
+            "https://i.postimg.cc/W36GhSFq/stol.png"
+        ]
+    }
+
+    // MARK: - Public properties
+
+    @Published var tagSelection: String?
+
+    // MARK: - Private Properties
+
+    @State private var urls = Constants.urls
```

Comment:
```suggestion
    @State private var urls = Constants.urls
```
А есть смысл писать  @State в классе?